README.md

```diff
@@ -1,3 +1,210 @@
+# terraform-equinix-template
+
+[![run-pre-commit-hooks](https://github.com/equinix-labs/terraform-equinix-metal-vcf/actions/workflows/pre-commit.yaml/badge.svg)](https://github.com/equinix-labs/terraform-equinix-metal-vcf/actions/workflows/pre-commit.yaml)
+[![generate-terraform-docs](https://github.com/equinix-labs/terraform-equinix-metal-vcf/actions/workflows/documentation.yaml/badge.svg)](https://github.com/equinix-labs/terraform-equinix-metal-vcf/actions/workflows/documentation.yaml)
+
+`terraform-equinix-metal-vcf` is a minimal Terraform module that utilizes [Terraform providers for Equinix](https://registry.terraform.io/namespaces/equinix) to provision digital infrastructure and demonstrate higher level integrations.
+
+<img src="assets/8-VCF_on_Metal_ft_Metal_VRF.png" width="608" height="552" alt="Target Metal Architecture featuring Metal VRF for Underlay routing">
+
+## Usage
+
+This project is supported by the user community. Equinix does not provide support for this project.
+
+Install Terraform using the [tfenv](https://github.com/tfutils/tfenv) utility.
+
+This project may be forked, cloned, or downloaded and modified as needed as the base in your integrations and deployments.
+
+This project may also be used as a [Terraform module](https://learn.hashicorp.com/collections/terraform/modules).
+
+To use this module in a new project, create a file such as:
+
+```hcl
+# main.tf
+terraform {
+  required_providers {
+    equinix = {
+      source = "equinix/equinix"
+    }
+}
+
+module "example" {
+  source = "github.com/equinix-labs/terraform-equinix-metal-vcf"
+  # Published modules can be sourced as:
+  # source = "equinix-labs/terraform-equinix-metal-vcf/equinix"
+  # See https://www.terraform.io/docs/registry/modules/publish.html for details.
+
+  # version = "0.1.0"
+
+  ## Metal Auth
+  metal_auth_token      = ""  # API Token for Equinix Metal API interaction https://deploy.equinix.com/developers/docs/metal/identity-access-management/api-keys/
```

Comment:
If we're not passing the `metal_auth_token` to deeper levels, we can omit the Terraform variable and ask the user to define this via environment variables, `METAL_AUTH_TOKEN` (`eval $(metal env --export)`).

Reply:
It's also used in the modules, can we remove it even though it's used there?